drivers/sensor/nooploop/tofsense/tofsense_can.h

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (c) 2025 Alex Fabre
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_DRIVERS_SENSOR_NOOPLOOP_TOFSENSE_CAN_H_
+#define ZEPHYR_DRIVERS_SENSOR_NOOPLOOP_TOFSENSE_CAN_H_
+
+#if !defined static_assert
+#define static_assert _Static_assert
+#endif
```

Comment:
```suggestion
#ifndef static_assert
#define static_assert _Static_assert
#endif
```
or
```suggestion
```
You can use the `ASSERT_*` series of macros. It seems to be called `STATIC_ASSERT`.

Reply:
Well, maybe I missed it, but I can't find any static assert macro, can you show me ?

Reply:
`BUILD_ASSERT`

Reply:
Updated, thanks